rpc-rs/src/provider_main.rs

```diff
@@ -152,12 +152,12 @@ where
     for ld in initial_links.into_iter() {
         if let Err(e) = provider_dispatch.put_link(&ld).await {
             eprintln!(
-                "Error starting provider: failed to initialize link {:?}",
-                &ld
+                "Error starting provider: failed to initialize link {:?}: {:?}",
+                &ld, e
```

Comment:
nit: Perhaps this would read better as "Failed to initialize link during provider startup" since we will still start the provider regardless